modules/Bio/EnsEMBL/Variation/DBSQL/vcf_config.json

```
@@ -1028,12 +1038,20 @@
       "filename_template" : "https://ftp.ensembl.org/pub/release-106/compara/conservation_scores/91_mammals.gerp_conservation_score/gerp_conservation_scores.pan_troglodytes.Pan_tro_3.0.bw",
       "annotation_type": "gerp"
     },
+    {
+      "id": "27_sauropsids.gerp_conservation_score",
+      "species": "coturnix_japonica",
+      "assembly": "Coturnix_japonica_2.0",
+      "type": "remote",
+      "filename_template" : "/27_sauropsids.gerp_conservation_score/gerp_conservation_scores.coturnix_japonica.Coturnix_japonica_2.0.bw",
```

Hi @likhitha-surapaneni! Everything looks good, but ensembl-variation we add the full URL to the conservation score files:

```suggestion
      "filename_template" : "https://ftp.ensembl.org/pub/release-106/compara/conservation_scores/27_sauropsids.gerp_conservation_score/gerp_conservation_scores.coturnix_japonica.Coturnix_japonica_2.0.bw",
```